x/gov/keeper/grpc_query.go

```
@@ -18,6 +18,13 @@ import (
 
 var _ v1.QueryServer = Keeper{}
 
+func (q Keeper) Constitution(c context.Context, req *v1.QueryConstitutionRequest) (*v1.QueryConstitutionResponse, error) {
```

nit: this does not need to return an error as no function returns an error